model/shared/src/main/scala/explore/model/NonEmptySetWrapper.scala

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
```

Comment:
I wonder if this file should go in `explore.model.util` to match the pattern in `lucuma.core`? The current `utils.scala` could be made a package object in the new package and references changed (to remove the `s`).

Reply:
Sounds good to me.